app/init.php

```diff
@@ -58,6 +58,7 @@
 const APP_LIMIT_ANTIVIRUS = 20971520; //20MB
 const APP_LIMIT_ENCRYPTION = 20971520; //20MB
 const APP_LIMIT_COMPRESSION = 20971520; //20MB
+const APP_CHUNK_SIZE = 2 * (1024 * 1024);
```

Comment:
I think we need two size here,
1. chunk size is the return chunk size which i set 2MB
2. chunking limit, the file size above which the download/upload endpoints will chunk and this right now I'm using the compression limit, because once we compress, chunking will not work as expected

Reply:
Is this the same limit we use for the Swoole `Response::send()` method?

Reply:
don't get what you mean. Response::send() does similar thing, any data above 2MB is chunked, so even if we read a 20 MB  file from storage and send using `Response::send()` as we do now, the response will still be in 2MB chunks.

But we need another limit for files when not to read them whole in memory, we can use that same limit for Compression as well, because we cannot compress/decompress without reading as a whole in the memory at the moment. Don't know yet if there might be other way.